assets/reviews.js

```diff
@@ -125,6 +125,224 @@ const setupReviews = async () => {
   }
 };
 
-document.addEventListener('DOMContentLoaded', () => {
+const reviewData = {
+  content: '',
+  email: '',
+  ratings: 0,
+  first_name: '',
+  last_name: '',
+  images: []
+};
+
+function setupEventListeners() {
+  const reviewForm = document.getElementById('reviewForm');
+  const modal = document.getElementById("reviewModal");
+  const btn = document.getElementById("addReviewBtn");
+  const span = document.querySelector(".close");
+  const starRadios = document.querySelectorAll('.star-rating input[type="radio"]');
+
+  reviewForm.addEventListener('submit', handleReviewFormSubmit);
+  btn.addEventListener('click', showModal);
+  span.addEventListener('click', hideModal);
+
+  starRadios.forEach(radio => {
+    radio.addEventListener('change', function() {
+      const starCountSpan = document.querySelector('.review-stars-count');
+      starCountSpan.textContent = `(${this.value} ${ratings})`;
+    });
+  });
+
   setupReviews();
-});
+}
+
+async function handleReviewFormSubmit(e) {
+  e.preventDefault();
+  clearFieldErrors();
+
+  const formData = new FormData(e.target);
+  Object.assign(reviewData, {
+    content: sanitizeInput(formData.get('content')),
+    email: formData.get('email'),
+    ratings: Number(formData.get('ratings')),
+    first_name: formData.get('first_name'),
```

Comment:
while you have create `sanitizeInput()` you can use it for first_name & last_name inputs